### DISCUSSION TITLE
Counter-Strike - HLSW Problems

### DISCUSSION THREAD

**Discussion by knuis52** [GameServerManagers/LinuxGSM#3562] [Q&A]
I have successfully installed and completed.
I can enter the game, no problem.
But from the outside, the game looks closed.
The game appears offline. 
The game is open but looks closed when viewed from the outside. [hlsw,gamebanana,gametracker, my server look OFFLINE]
But when I do a normal installation, without linuxgsm, the server appears to be online (hlsw).

**Comment by visdmin** [comment]
Hello,

Would it be possible for you to post your `csgo` server configuration `csgoserver.cfg` ?
**PLEASE** remember to censure any sensitive information !

You can find it in the following directory:
`[installation directory]/lgsm/config-lgsm/csgo/csgoserver/csgoserver.cfg`

I am wondering if this could have somethings to do with the `gslt` token.
https://docs.linuxgsm.com/steamcmd/gslt
